zerver/openapi/python_examples.py

```diff
@@ -1183,7 +1293,7 @@ def update_message(client: Client, message_id: int) -> None:
         url=url,
         method="GET",
     )
-    assert result["result"] == "success"
+    validate_response_result(result)
     assert result["raw_content"] == request["content"]
```

Comment:
There are three places that we call `GET messages/{message_id}` to compare the `raw_content` of the message with the request message `content`. Seems like another place we could add a helper function.

Reply:
Created the `validate_message` helper function.